### DISCUSSION TITLE
multiple jpa to utilize storage-area

### DISCUSSION THREAD

**Discussion by Fex97** [keycloak/keycloak#14160] [Archive]
Hello!
I have looked around and tried your example and just wondering if it is planned  to be able to use several postgres db's and split storage-area between them instead of just on jpa + infinispan etc.
This discussion also mention something of it: https://github.com/keycloak/keycloak/discussions/10324
Thanks

**Comment by vramik** [comment]
Yes, you'd be able to set as many postgres databases as the number of the areas.
